### DISCUSSION TITLE
SHOW is already marked seen

### DISCUSSION THREAD

**Discussion by khunalex** [Flexget/Flexget#3428] [Q&A] [answered]
How can a show be marked as unseen?
I deleted it from the database using:
* flexget series forget SHOW
* flexget series remove SHOW
* flexget database clean  (It returned delete database without episodes)

running:
flexget series list -> the show is not there anymore

And despite of all of this, when I try to download it again, it keeps saying "SHOW is already marked as seen in the task"
What am I doing wrong? or is it a bug?

**Comment by khunalex** [comment, marked as answer]
disable:
   -seen

makes the trick

**Comment by tubedogg** [comment]
Run `flexget seen search SHOW`
Then you can use `flexget seen forget SEEN_TITLE` where `SEEN_TITLE` is the exact title as listed in the `seen search` results.

Although theoretically `flexget series forget SHOW` should have done that in the first place, if the entry wasn't associated with the series record, it's possible it may not have found it.

Also running both `series forget` and `series remove` are redundant. `forget` removes it from both seen and series; `remove` just removes it from series.
